framework/src/modules/interoperability/mainchain/commands/message_recovery.ts

```diff
@@ -156,4 +161,153 @@ export class MainchainMessageRecoveryCommand extends BaseInteroperabilityCommand
 			await this.internalMethod.addToOutbox(context, ccmChainId, newCcm);
 		}
 	}
+
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-expect-error
+	private async _applyRecovery(context: CrossChainMessageContext): Promise<void> {
```

Comment:
I didn't understand this comment. Do you mean add `def applyRecovery(trs: Transaction, ccm: CCM)` in the comment? I dont think it will help understand the logic.
I think It's better this command file is referenced to the whole command section, but it can be done in #7727